generated/_task_modules/channels/ai_channel.py

```diff
@@ -0,0 +1,202 @@
+# Do not edit this file; it was automatically generated.
+
+import ctypes
+import numpy
+
+from nidaqmx._lib import (
+    lib_importer, wrapped_ndpointer, ctypes_byte_str, c_bool32)
+from nidaqmx.scale import Scale
+from nidaqmx.errors import (
+    check_for_error, is_string_buffer_too_small, is_array_buffer_too_small)
+from nidaqmx._task_modules.channels.channel import Channel
+from nidaqmx.utils import unflatten_channel_string
+
+
+class AIChannel(Channel):
+    """
+    Represents one or more analog input virtual channels and their properties.
+    """
+    __slots__ = []
+
+    def __repr__(self):
+        return 'AIChannel(name={0})'.format(self._name)
+
+
+    @property
+    def ai_atten(self):
+        """
+        Specifies the amount of attenuation to use.
```

Comment:
The existing code indicates the type returned. I don't think that is a Python convention, but it does seem useful.

Reply:
BTW, Python has multiple docstring conventions. nidaqmx-python seems to use Google style docstrings, and including the return type for property getters seems to be a standard part of that convention: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google

Reply:
Another reason to add the type: generating exactly the same code makes it easier to diff the output. Once we're generating the same output, then we can start making changes to the output.

Reply:
I have added helpers methods to specify the return type as part of the documentation. As of now for enums/classes, only the name of the object will be added in the description not the entire path. For example, the expected docstring is `class: nidaqmx.system.scale.Scale: documentation` but currently documentation will look like `class: Scale: documentation`. We are yet to add the part of dynamically adding the import path of enums/classes, this will be added in consecutive PRs.